modules/content.yml

```diff
@@ -1,22 +1,25 @@
 name: content
 description: >-
-  Use markdown, JSON, YAML, XML or CSV in your nuxt application to easily write
-  content fetched using a MongoDB like API.
+  Nuxt Content reads the content/ directory in your project, parses .md, .yml, .csv or .json files and creates a powerful data layer for your application. 
+  Bonus, use Vue components in Markdown with the MDC syntax.
 repo: nuxt/content#main
 npm: '@nuxt/content'
-icon: content.png
+icon: nuxt.png
 github: https://github.com/nuxt/content
 website: https://content.nuxtjs.org
 learn_more: ''
 category: CMS
 type: official
 maintainers:
-  - name: Benjamin Canac
-    github: benjamincanac
-    twitter: benjamincanac
+  - name: Yaël Guilloux
+    github: Tahul
+    twitter: yaeeelglx
+  - name: Ahad Birang
+    github: farnabaz
+    twitter: a_birang
   - name: Sebastien Chopin
     github: Atinux
     twitter: Atinux
 compatibility:
-  nuxt: ^2.0.0
+  nuxt: ^2.0.0 || ^3.0.0
```

Comment:
Currently, i see readme and docs lack a reference to Nuxt 2 support (using content v1). Would be worth adding it at the same time of switching domain.

Reply:
Updated: https://github.com/nuxt/content#nuxt-2